### PR TITLE
Add Linux Support

### PR DESCRIPTION
**Pull request by akionsight**
This adds linux support to this application

the readme has also been updated for instructions for linux

### CODE REVIEW

**Comment by akionsight** [comment]
Thanks for your blog, helped me a lot

**Comment by akionsight** [comment]
also update the repo name and description if this PR is accepted

**Comment by akionsight** [comment]
Limitations: Currently only works on X11, working on a patch to enable wayland support

**Comment by roligheten** [comment]
Thanks for the PR, I will have a look at this once you have made the changes :slightly_smiling_face:

**Comment by akionsight** [comment]
> Limitations: Currently only works on X11, wnorking on a patch to enable wayland support

Ive been trying to get this working on Wayland for days now, With no success; the library I was using to do so (ydotool) requires super user permissions too which i would never give to such a quality of life program.

TL;DR: No wayland support as of now

**Comment by akionsight** [comment]
You may review the PR now, sorry for the delay
